docs/pre-release/releaseNotes.yml

```diff
@@ -41,13 +41,32 @@ items:
   - version: 2.4.8
     date: "TBD"
     notes:
+      - type: feature
+        title: VPN diagnostics tool
+        body: >-
+          There is a new subcommand, <code>test-vpn</code>, that can be used to diagnose connectivity issues with a VPN.
+          See the VPN docs for more information on how to use it.
+        docs: reference/vpn
       - type: feature
         title: RESTful API service
         body: >-
           A RESTful service was added to Telepresence, both locally to the client and to the <code>traffic-agent</code> to
           help determine if messages with a set of headers should be consumed or not from a message queue where the
           intercept headers are added to the messages.
         docs: reference/restapi
+
+      - type: change
+        title: TELEPRESENCE_LOGIN_CLIENT_ID env variable no longer used
+        body: >-
+          This value was configurable but there was no real reason to set it to be something different,
+          so that value is no longer used.
```

Comment:
"You could previously configure this value, but there was no reason to change it, so the value was removed."